.github/workflows/build.yml

```diff
@@ -216,6 +223,13 @@ jobs:
             ./crates/radix-engine-toolkit-uniffi/src/radix_engine_toolkit_uniffi.udl \
             --out-dir $OUTPUT_DIRECTORY \
             --lib-file $DYNAMIC_LIBRARY_PATH
+
+          cargo run \
```

Comment:
I think that we have some things missing in the build process. It looks like we currently generate the bindings and then upload the bindings file and that's about it, we do not generate a go package that bundles the code and the dynamic libraries together nor do we publish to any kind of package manager. 

I think that having a bundle of code and dynamic libraries (a package of sorts) is a must have as this is where some issues could happen. Having it be published to some kind of go package manager is a nice to have for other non-internal clients who wish to use the go library.

This workflow file has the process we follow to bundle up the dynamic libraries with the code of multiple languages.

Reply:
Summary of my research regarding Go package manager:
https://radixdlt.atlassian.net/wiki/spaces/~633a994d61dbef2805c10cb4/pages/3369795590/RET+Golang+binding#Release-process
Spoiler: there is none